lms/djangoapps/course_home_api/outline/v1/tests/test_views.py

```diff
@@ -35,6 +38,16 @@ def test_get_authenticated_enrolled_user(self, enrollment_mode):
         response = self.client.get(self.url)
         self.assertEqual(response.status_code, 200)
 
+        course_goals = response.data.get('course_goals')
+        goal_options = course_goals['goal_options']
+        if enrollment_mode == CourseMode.VERIFIED:
+            self.assertEqual(len(goal_options), 0)
+        else:
+            self.assertEqual(len(goal_options), 4)
```

Comment:
@mikix I don't feel like it's _great_ to base this test on length, but as of right now there are only 4 hardcoded goals... Now I'm thinking maybe I could change this to `self.assertTrue(len(goal_options) > 0)`? 🤔Let me know what you think

Reply:
Yeah, I like the idea of just checking that it's not zero. `self.assertGreater(len(goal_options), 0)` if you want to use unittest's weirdly specific assert methods.